src/Terrain/TODO.txt

```diff
@@ -0,0 +1,2 @@
+The src floder is getting to bloated. terrain file, and things that the terrain file inherits should be moved into 
```

Comment:
```suggestion
The src floder is getting too bloated. terrain file, and things that the terrain file inherits should be moved into 
```

Reply:
I want to do this when there are less branches